nifxml.py

```diff
@@ -719,7 +755,7 @@ def member_name(n):
     @rtype: string
     """
     if n == None: return None
-    if n == 'ARG': return 'ARG'
+    if n.startswith('ARG'): return n
```

Comment:
I know you're just porting over figment's changes, but this is no longer necessary as the `ARG[]` syntax is no longer used in nifxml.  Though leaving it is also fine, it's just nifxml won't use `ARG[]` syntax for anything until NifSkope's expression parser can support it.  Which won't happen until there's a need for the syntax again.